week2/homework/src/index.js

```diff
@@ -1,3 +1,82 @@
-'use strict';
+'use strict:';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+const { promisify } = require('util');
+const dataToNewLine = data => `\n${data}`;
+
+const writeFile = promisify(fs.writeFile);
+const readfile = promisify(fs.readFile);
+
+const filePath = './ToDoList.json';
+
+const getAllToDos = () => {
```

Comment:
👍 nice to put this in a separate function

Reply:
👍